spa/src/App.vue

```diff
@@ -35,9 +35,16 @@
       <div class="text-clock text-center w-full py-0.5">
         <ClockPage />
        </div>
+        <div class="flex justify-center w-full pb-5 cursor-pointer">
+          <div>
+            <center>
+              <span class="underline" style="color: yellow;" @click="openCitizenOnlineModal">Citizens Online</span>
+              <button class="btn-ui" @click="callGuide"><font color='lime' size="1.5rem">Call a Guide</font></button>
```

Comment:
This was in the citizen online page.  I fear an abuse of the button if it is that accessible.

Reply:
This is where it was on ct. I can change it to within the online citizens list if that would be better.

Reply:
Ahh... found that photo too.  Must have been a late change as I have other photos that don't have it.  Might be a good thing to hide for now until we have City Guides.

Reply:
Will do